src/client/debug/debug.proto

```diff
@@ -37,6 +37,7 @@ message BinaryRequest {
 
 message DumpRequest {
   Filter filter = 1;
+  int64 limit = 2;
```

Comment:
This limits the number of commits and jobs returned right? I think maybe a slightly more specific name than limit would be good. Just calling it `limit` makes it seem like it's a global limit on the number of bytes returned.

Reply:
Yeah, I wasn't really sure what to call this considering it applies to commit and job counts. I actually don't have a good name in mind because `commit_and_job_limit` seems more confusing and splitting it up into two fields seems like it would decrease the ease of use because it seems unlikely that a user will want to have limits that are specific to the commits and jobs. I think `limit` might be fine if I just add a note in the proto (the cmd already has a description that I think is adequate).

Reply:
Yeah a note in the protos should be fine. I don't have a better name either.